test/functional/options/defaults_spec.lua

```diff
@@ -328,3 +331,277 @@ describe('XDG-based defaults', function()
     end)
   end)
 end)
+
+
+describe('stdpath()', function()
+  -- Returns an OS friendly version of the unix_path.
+  local function alter_path(unix_path)
+    if helpers.iswin() then
+      return string.gsub(unix_path, '/', '\\')
```

Comment:
Also `string` is not needed, it is a method of all strings: `unix_path:gsub('/', '\\')`.